cogs/text_service_cog.py

```diff
@@ -693,6 +693,7 @@ async def ask_command(
         self,
         ctx: discord.ApplicationContext,
         prompt: str,
+        ephemeral: bool,

```

Comment:
Can we rename this to be `hidden`, `ephemeral` might be too weird sounding for some people